### DISCUSSION TITLE
Websockets with libcurl

### DISCUSSION THREAD

**Discussion by Susheelabhat** [curl/curl#10262] [Q&A] [answered]
Hi. I'm trying to use websockets with libcurl. Can I hit the "wss" endpoint directly ? I can't find an example which shows how to connect to a wss URL.

This is what I want to hit.
 curl_easy_setopt(easy, CURLOPT_URL, "wss://");
It gives ,
*** Protocol "wss" not supported or disabled in libcurl
* Closing connection -1** even when using latest libcurl version and comipiling it with websocket enabled.

Is it possible to get an example which demonstrates how to send and receive data directly to a wss url?

**Comment by bagder** [comment, marked as answer]
That means you did not build curl WebSocket enabled. Since it is still experimental, you need to enable it in your build. 

If you use configure: `./configure --enable-websockets`
